dts/riscv/rapidsilicon/rapidsi_virgo.dtsi

```diff
@@ -116,7 +112,7 @@
 			 interrupts = <22 8>;
 			 interrupt-parent = <&eclic>;
 			 clock-frequency = <266666667>;
-			 status = "okay";
+			 status = "disabled";
```

Comment:
I know you mentioned somewhere that the driver / KConfig enables this somewhere. But I dont think that should be the case. This should be enabled by default from the hardware, especially something like UART. Should not depend on Kconfig or the driver. Lets look at all the disables and okays in the peripherals

Reply:
> Basically, the drivers are enabled using DTS. Kconfig only enables the options which are software related. Kconfig works like compiler preprocessor macros etc. But, if a driver is dependent on some other driver, then we need to enable that in menuconfig. Like, if we have to use PVT, we can let it be status=okay in our DTS, but it also needs Sensors to be enabled and that has to be done via menuconfig.

RIght, but so UART shouldnt UART be okay by default in dts? Here we changed it to disabled. What sensors, for example, does UART need apart from the UART controller?

Reply:
Yes, that we can keep it to be okay by default for all kinds of boards. That is no problem.